test/api/importcsv/api_test.go

```diff
@@ -0,0 +1,166 @@
+package importcsv
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/binding"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestImportCSV(t *testing.T) {
+	for _, r := range TestCases {
+		t.Run(r.FileName, func(t *testing.T) {
+
+			// Upload CSV.
+			inputData := api.ImportSummary{}
+			assert.Must(t, Client.FilePost(api.UploadRoot, r.FileName, &inputData))
+
+			// Since uploading the CSV happens asynchronously we need to wait for the upload to check Applications and Dependencies.
+			time.Sleep(time.Second * 3)
+
+			// Check list of Applications.
+			gotApps, _ := Application.List()
+			expectedApps := r.ExpectedApplications
```

Comment:
Not needed declaration of expectedApps, why don't use r.ExpectedApplications directly? (commented the same thing few lines below few days back..)

Reply:
Sorry just because it was a long name thatswhy